render/src/simple_element.rs

```diff
@@ -3,7 +3,7 @@ use crate::html_escaping::escape_html;
 use std::collections::HashMap;
 use std::fmt::{Result, Write};
 
-type Attributes<'a> = Option<HashMap<&'a str, &'a str>>;
+type Attributes<'a> = Option<HashMap<&'a str, String>>;
```

Comment:
I'm not sure why is this change needed — but I'm new to Rust. can you give me an example where `&'a str` isn't good enough?

Maybe for _returning_ a component other than rendering it?

Reply:
I've encountered a few cases where it would be useful to pass owned values as attributes, but this change would cause unnecessary cloning. Maybe a Cow would be a good idea?

Reply:
+1 undo this change since you're handling it in #21.